src/ComputeSharp.Graphics/Buffers/ConstantBuffer{T}.cs

```diff
@@ -33,7 +33,7 @@ public sealed class ConstantBuffer<T> : HlslBuffer<T> where T : unmanaged
         private static int GetPaddedSize()
```

Comment:
I guess we should hint this should always be inlined, correct?

```suggestion
        [MethodImpl(MethodImplOptions.AggressiveInlining)]
        private static int GetPaddedSize()
```

Reply:
It will always be yep, annotated it on the commit i'm about to push

Reply:
Fixed in https://github.com/Sergio0694/ComputeSharp/pull/68/commits/c613d92413a8e45a044032e9759351ba5b305fa3.